code/modules/cargo/packs/organic.dm

```diff
@@ -329,7 +329,7 @@
 /datum/supply_pack/organic/seeds
 	name = "Seeds Crate"
 	desc = "Big things have small beginnings. Contains fourteen different seeds."
-	cost = 1250
```

Comment:
Should lable this all, cuz ~~im~~ people upstream are going to be tweaking these